capstone/src/main/java/com/google/step/data/RestaurantScore.java

```diff
@@ -0,0 +1,46 @@
+package com.google.step.data;
+
+/**
+ * Class that holds the score for a specific restaurant.
+ */
+public class RestaurantScore {
+
+    // The name should always remain the same but the score will change when calculating it.
+    private final String restaurantName;
```

Comment:
Is this being used to uniquely identify a restaurant? If so, consider using the restaurant key since that's guaranteed to be unique.

Reply:
You're right. Changed!